docs/reference/how-to/knn-search.asciidoc

```diff
@@ -66,6 +71,14 @@ reduction techniques like PCA. When experimenting with different approaches,
 it's important to measure the impact on relevance to ensure the search
 quality is still acceptable.
 
+[discrete]
+=== Reduce vector memory foot-print
+
+As of 8.12 the default <<dense-vector-element-type,`element_type`>> is `float`. But this can be
+automatically quantized during index time through the <<dense-vector-quantization,`quantization`>>. Quantization will
+reduce the required memory by 4x, but it will also reduce the precision of the vectors. For `float` vectors with
```

Comment:
Can we link to any information (blog maybe?) here that can explain the space/precision tradeoffs?